src/ray/util/container_util.h

```diff
@@ -28,56 +30,92 @@
 namespace ray {
 
 template <typename T>
-std::string debug_string(const T &obj) {
-  std::stringstream ss;
-  ss << obj;
-  return ss.str();
+class DebugStringWrapper;
+
+// The actual interface.
+template <typename T>
+DebugStringWrapper<T> debug_string(const T &t) {
```

Comment:
Same here, prefer `SnakeCase` if we follows google coding style

Reply:
for this function, we intended to mimic std library functions so we used camel_case. This also saves us from changing all callsites.